packages/ui/src/lib/dropdown/Dropdown.svelte

```diff
@@ -192,6 +191,7 @@ function onWindowClick(e: Event): void {
     disabled={disabled}
     id={id}
     name={name}
+    aria-label={ariaLabel}
```

Comment:
why the aria-label attribute is moved ?

Reply:
I don't think we need an aria-label for the div, only for the button for testing purposes. The aria-label on the div seems unnecessary.

Reply:
Do you have some test code this is trying to fix or is it just an aria lookup checking the text content? There is one case in CreateVolume using userEvent.selectOptions(), but this change (alone) would not fix it.

Reply:
My second proposal is to add an aria-label to both the div and the button elements. Button will have `ariaLabel={`${aria-label} Button`}`

Reply:
Does the button's div have any function other than a container? I am wondering why the container div of a button should have aria-label at all. It makes sense for some elements with role, ie. this button. Unless there is some use case.